Preparing an extended version of the changelog for the upcoming release (#1521)
skrub-data/skrub: CHANGES.rst

Ongoing development
===================

Highlights
------------

> - The skrub DataOps are now available! DataOps represent a powerful new way of

Reviewer: ```suggestion
- Major feature! Skrub DataOps are a powerful new way to
```

- The :class:`TableReport` now shows if columns are sorted. :pr:`1512` by :user:`Dea María Léon<DeaMariaLeon>`.

Bugfixes
--------
- Fixed a bug that caused the :class:`StringEncoder` and :class:`TextEncoder` to raise an exception if the
  input column was a Categorical datatype. :pr:`1401` by :user:`Riccardo Cappuzzo<rcap107>`.

> Documentation
> ---------

Reviewer: ```suggestion
Documentation
-------------
```
